vendor-extra/ContentPageBundle/src/Controller/ContentController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Libero\ContentPageBundle\Controller;
+
+use Symfony\Component\HttpFoundation\Response;
+
+final class ContentController
+{
+    public function __invoke(string $id) : Response
+    {
+        return new Response($id, Response::HTTP_OK, ['Content-Type' => 'text/plain']);
```

Comment:
Next step is to use a client to request a content `front` (which should just be static/faked data for now).